apis/cloudfront/v1alpha1/zz_types.go

```diff
@@ -1112,35 +1204,41 @@ type RealtimeLogConfigs struct {
 	NextMarker *string `json:"nextMarker,omitempty"`
 }
 
+// +kubebuilder:skipversion
 type Restrictions struct {
 	// A complex type that controls the countries in which your content is distributed.
 	// CloudFront determines the location of your users using MaxMind GeoIP databases.
 	GeoRestriction *GeoRestriction `json:"geoRestriction,omitempty"`
 }
 
+// +kubebuilder:skipversion
 type S3Origin struct {
 	DomainName *string `json:"domainName,omitempty"`
 
-	OriginAccessIDentity *string `json:"originAccessIDentity,omitempty"`
+	OriginAccessIdentity *string `json:"originAccessIdentity,omitempty"`
 }
 
+// +kubebuilder:skipversion
 type S3OriginConfig struct {
-	OriginAccessIDentity *string `json:"originAccessIDentity,omitempty"`
+	OriginAccessIdentity *string `json:"originAccessIdentity,omitempty"`
```

Comment:
This and the above will be breaking changes. It's an alpha API so we can do that, but it's probably worth a prominent release note at least.